File: coding_freshmen/JAVA/MAJORITY_ELEMENT/MAJORITY_ELEMENT.java
public class Main {
    public static void main(String[] args) {

        int[] nums = {7,1,5,7,8,12,24,7,7,7};
        System.out.println(majorityElement(nums));
    }
    static int majorityElement(int[] arr){
        int count = 0, max = 0, temp = -1;
        for (int i = 0; i < arr.length; i++) {
            for (int j = 0; j < arr.length; j++) {
                if (arr[i] == arr[j]){
                    count++;
                    if (count > max){
                        max = count;
                        temp = arr[i];
                    }
                }
            }
            count = 0;
        }
        return temp;
    }
}

File: coding_freshmen/PYTHON/JYOTISHREE_BEHERA/MAJORITY_ELEMENT.py
# function declaration to find the majority element
def majority_element(arr,n):

    for i in set(arr):                 # set is used to count an element once 
        if (arr.count(i)) > (n//2):     # if count of an element is greater than n/2 return the corresponding element
            print(i)
                
# input an array of single spaced integers
nums = list(map(int,input().split()))
n = len(nums) # n is the length of array
majority_element(nums,n)